### PR TITLE
Implement SoundCloud support

### LINKED ISSUE #2372: Implement SoundCloud support
**Actual Behaviour**

Not present, soundcloud support was recently integrated in server side

**Expected Behaviour**

Implement SoundCloud support in chat

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #2372 

Changes: Integrate SoundCloud support in Chat App.

Demo Link: https://pr-2379-fossasia-susi-web-chat.surge.sh

Screenshots of the change:

Screenshot 2019-06-26 at 10 33 51 PM

### CODE REVIEW

**Comment by fragm3** [comment]
@akshatnitd @harsh-jindal please have a look

**Comment by harsh-jindal** [comment]
@fragm3 Keep the song on auto play

File: src/components/ChatApp/MessageListItem/generateMessageBubbleUtil.js [hunk 1]
```diff
@@ -231,11 +231,21 @@ const generateAudioBubble = (
...
-            src={src}
-            title="SUSI features audio"
-            frameBorder="0"
-            allowFullScreen
-          />
+            id="sc-widget"
+            title="SUSI Soundcloud"
+            src={`https://w.soundcloud.com/player/?url=${src}`}
+            width="100%"
+            height="20rem"
```
Comment (akshatnitd) [inline src/components/ChatApp/MessageListItem/generateMessageBubbleUtil.js]: This way the height is not applied in rem. Use style to apply height in rem. Also, increase the height than the current one.

File: src/components/ChatApp/MessageListItem/generateMessageBubbleUtil.js [hunk 2]
```diff
@@ -231,11 +231,21 @@ const generateAudioBubble = (
       <MessageBubble author={message.authorName}>
         <div>
```
Comment (akshatnitd) [inline src/components/ChatApp/MessageListItem/generateMessageBubbleUtil.js]: Add margin-top: 10px

**Comment by fragm3** [comment]
@akshatnitd @harsh-jindal made the changes, it autoplays now:

Screenshot:

Screenshot 2019-06-27 at 1 12 03 AM

File: src/components/ChatApp/MessageListItem/generateMessageBubbleUtil.js (hunk 1 above)
Comment (fragm3) [reply src/components/ChatApp/MessageListItem/generateMessageBubbleUtil.js]: Fixed it

**Comment by fragm3** [comment]
If I increase the size `Cookie Policy` appears and it can't be removed or hidden.

**Pull request merged**